addons/vela-core-shard-manager/README.md

```diff
@@ -0,0 +1,10 @@
+# vela-core-shard-manager
+
+The vela-core-shard-manager is a manager for slave application controllers when sharding is enabled.
+
+## install
+
+```shell
+vela addon enable vela-core-shard-manager nShards=3
+```
+
```

Comment:
```suggestion
## Why using addon for sharding?
```

Please explain the benifits

Reply:
Concerns:

1. does the addon need  special permission?
2. the stability and security, will the addon be wrongly removed and affected the core?

Reply:
Done